src/js/media-theme-element.ts

```diff
@@ -189,26 +198,26 @@ export class MediaThemeElement extends globalThis.HTMLElement {
   }
 
   #updateTemplate(): void {
+    if (this.#template) return;
```

Comment:
why is this added? 
what if one first sets a node via template prop and after it sets a string?